bp-worlmap v11 & fixes for map icon with bg2

TotoR115 opened the pull request:
'Update for compatibility with bp-worldmap v11 :
- map icon for YS0390.are is 182
- map icon for YS0370.are is 98
- moove YS0390 up, as it is with Bg2

Fixes:
- the mod will now install the new map icon with bg2
- bgt map icon's position should now work
- update add_worldmap_tbl.tpa
- remove useless script (ys0407.bcs) to ys0407.are. This was not required as the script is already triggered by a trigger...

TotoR115 committed:
Miscellaneous fixes

- YS1001.baf: use existing movies, as Resources don't exist
- Add missing sound
- update patches.tpa
- ys0040.are: is missing the ys0040.bcs
- ys0160.are: drawers x: 737;Y: 379 does not exists
- ys0161.are: Container 1 is anormaly empty. Add MISC07.ITM like others
- ys0401.are: Chest contains not existing itm
- ys0407.are: is missing the ys0407.bcs
- yshldgrd.cre: remove the shield as in oBG2 he can't use his bow

TotoR115 committed:
Update BWFISHW.d

Small typo, RES linked to Fisher's wife dialog in Porthpentyrch

TotoR115 committed:
Merge branch 'InfinityMods:master' into master

TotoR115 committed:
bp-worlmap v11 & fixes for map icon with bg2

Update for compatibility with bp-worldmap v11 :
- map icon for YS0390.are is 182
- map icon for YS0370.are is 98

Fixes:
- the mod will now install the new map icon with bg2
- bgt map icon's position should now work
- update add_worldmap_tbl.tpa
- remove useless script (ys0407.bcs) to ys0407.are. This was not required as the script is already triggered by a trigger...

TotoR115 committed:
Update areas.tbl

position update for bp-worldmap

GraionDilach commented:
Shouldn't the Government District Westside be revealed with Flag 3?

From what I noticed, Athkatla city areas are revealed by default, but I don't know the content of this mod.

TotoR115 committed:
adjuste position of YS0390

- the map icon position of YS0390.are was too low on Worldmap from bp-worlmap mod in comparison with worldmap from BGT and BG2.

TotoR115 commented:
> Shouldn't the Government District Westside be revealed with Flag 3?

Don't think I will change that flag, as it seem to be intended this way (flag 0).
